cmd.go

```diff
@@ -53,14 +53,14 @@ func run(r io.Reader, w io.Writer, args []string) error {
 	handler := transfer.NewPktline(r, w, logger)
 	for _, cap := range capabilities {
 		if err := handler.WritePacketText(cap); err != nil {
-			logger.Logf("error sending capability: %s: %v", cap, err)
+			logger.Log("error sending capability", "cap", cap, "err", err)
```

Comment:
For structured logging, you may want to consider using a `map[string]interface{}` instead, to integrate the structure into loggers that use JSON. For example:

```go
logger.Log("error sending capability", map[string]interface{}{"cap": cap, "err": err})
```

Reply:
I like the simplicity of `Log(msg string, kv ...interface{})`, implementors can convert the key to string using `fmt.Sprint(k)`

```go
fields := map[string]interface{}{}
for i := 0; i < len(kv); i += 2 {
    fields[fmt.Sprint(kv[i])] = kv[i+1]
}

```